### PR TITLE
Scc endpoints

### PR DESCRIPTION
**Pull request by jesusbv**
## Description

The endpoints SUMa needs on RMT must be the same path as SCC

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [X] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [X] I have verified that my code follows RMT's coding standards with `rubocop`.
- [X] I have reviewed my own code and believe that it's ready for an external review.
- [X] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [X] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/320966/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged b3ceccd9f10667641d3cd3595a99ec46d4ffe63f into 4232f5c99611c0615ac908afe99393f841bea4f6)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/320967/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged e488b714a930c598150482bf7f6be4b0e9ddd47a into 4232f5c99611c0615ac908afe99393f841bea4f6)

**Review by rjschwei — APPROVED** [review]

File: config/routes.rb [hunk 1]
```diff
@@ -62,4 +62,10 @@
   mount RegistrationSharing::Engine, at: '/api/regsharing' if defined?(RegistrationSharing::Engine)
   mount InstanceVerification::Engine, at: '/api/instance' if defined?(InstanceVerification::Engine)
   mount SccSumaApi::Engine, at: '/api/scc' if defined?(SccSumaApi::Engine)
+
+  get '/connect/organizations/products/unscoped', to: redirect(path: '/api/scc/unscoped-products')
```
Comment (digitaltom) [inline config/routes.rb:66]: Is it possible to move all of this  and the `mount`s above into a `if defined?(SccSumaApi::Engine)` block?

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/321266/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 0d8528b1045a59d4af17b8673fd1e407e8e0297e into 4232f5c99611c0615ac908afe99393f841bea4f6)

**Pull request merged**